demos/backup-la-reporting/azuredeploy.json

```diff
@@ -2,12 +2,12 @@
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {
-      "location": {
+      "resourceGroupName": {
```

Comment:
remove the resourceGroup parameter here and throughout - also in the prereq template... it's never really used to change the resourceGroup from the deployment itself.